check_translations.sh

```
@@ -0,0 +1,22 @@
+DEFAULT_KEYS="default_keys.txt"
+for file in $(find app/src/main/res -name "strings.xml" | grep "values-"); do
+    echo "Checking $file..."
+    grep "<string name=" "$file" | sed -E 's/.*name="([^"]+)".*/\1/' | sort > current_keys.txt
+    
+    # Check for ExtraTranslation
+    EXTRA=$(comm -23 current_keys.txt "$DEFAULT_KEYS")
+    if [ ! -z "$EXTRA" ]; then
+        echo "  [!] EXTRA KEYS (will break build):"
+        echo "$EXTRA" | sed 's/^/    - /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)